### PR TITLE
add p7zip to test tools

### PR DESCRIPTION
**Pull request by gdams**
required for macOS codesigning

### CODE REVIEW

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/MacOSX.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@ Test_Tool_Packages:
   - mercurial
   - pulseaudio
   - cpanminus
+  - p7zip
```
Comment (karianna) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/MacOSX.yml]: Reason?  Add a comment please

**Comment by gdams** [comment]
@karianna can this be merged now?

**Comment by sxa555** [comment]
@gdams You haven't added the requested comment as far as I can tell

**Comment by gdams** [comment]
done

**Comment by sxa555** [comment]
Out of interest, why did we change to use `7z` instead of regular `zip`/`unzip`?
Ref: https://github.com/AdoptOpenJDK/openjdk-build/commit/ec5401f71e290f6b8d788daeaf2506cfda7de95c

**Comment by gdams** [comment]
@sxa555 the built in macOS unzip cannot handle things zipped with PKZIP. See https://unix.stackexchange.com/questions/115825/extra-bytes-error-when-unzipping-a-file

**Comment by sxa555** [comment]
Interesting - are the files initially being created with `pkzip` then? Haven't seen that used in years. Maybe add a comment `Used by signing because unzip can't handle pkzip file-created files`

**Pull request merged**